docs/_pages/extensibility.md

```diff
@@ -36,6 +36,7 @@ public class DirectoryInfoAssertions :
 
     protected override string Identifier => "directory";
 
+    [CustomAssertion]
```

Comment:
🔧 I think we should at least explain what this attribute does in the text following the example.

Reply:
@dennisdoomen Do you think it is a good idea to duplicate the explanation fron Introduction? https://github.com/fluentassertions/fluentassertions/blob/180c1a33aa41538674fad623c37af4bb20c586ab/docs/_pages/introduction.md?plain=1#L142

Reply:
Just a summary of why it is needed.

Reply:
@dennisdoomen 

This good https://github.com/fluentassertions/fluentassertions/pull/2685/commits/9bebe76c4c7ce89ffed89c33835842ff5def68ef? Or is it too vague and summarized, and I should elaborate more on in what way test fail messages are more meaningful with `[CustomAssertion]` than without it?